ops_boba/monitor/services/monitoring.js

```diff
@@ -10,6 +13,15 @@ let l1GasPrice
 let l2PoolBalance
 let l2BlockNumber
 let l2GasPrice
+const oracleAddresses = []
+const l1Provider = new ethers.providers.JsonRpcProvider(configs.l1Url)
```

Comment:
Could you please use `StaticJsonRpcProvider `?

Reply:
Looks like this was applied